modules/mobile/docs/schemas/Prescriptions.yml

```diff
@@ -16,14 +16,21 @@ properties:
       prescriptionStatusCount:
         type: object
         description: |
```

Comment:
@kpethtel I ended up leaving this as a single block because I felt doing each one individually would start being repetitive (except for active and isRefillable)